### DISCUSSION TITLE
Atlantis Primeng Angular 17 SSR

### DISCUSSION THREAD

**Discussion by ronaldcorreagane** [primefaces/community#1966] [PrimeNG Templates]
The Atlantis primeng template does not work correctly with Angular 17 default Server-side rendering SSR. Is there a solution? When would it be available?

**Comment by Haikelano** [comment]
the same issue here, i trying different why buit the same result. 
I think the main probleme here is the archetecture of the components.
i hope in next version we got a big upgrade to handle this issue.
